File: app/src/main/java/com/diwixis/filmlibrary/Params.java
package com.diwixis.filmlibrary;

import java.util.HashMap;

/**
 * Created by Diwixis on 19.04.2017.
 */

public class Params {
    public static HashMap<String, String> getMovieParams(){
        HashMap<String, String> hashMap = new HashMap<>();
        hashMap.put("api_key", Constants.apiKey);
        hashMap.put("language", "en");
        hashMap.put("page", "1");
        return hashMap;
    }
}

File: app/src/main/java/com/diwixis/filmlibrary/Params.kt
package com.diwixis.filmlibrary

import java.util.*

/**
 * Created by Diwixis on 19.04.2017.
 */
object Params {
    val movieParams: HashMap<String, String>
        get() {
            val hashMap = HashMap<String, String>()
            hashMap["api_key"] = BuildConfig.API_KEY
            hashMap["language"] = "en"
            hashMap["page"] = "1"
            return hashMap
        }
}